autokeras/hypermodel/hyper_head.py

```diff
@@ -17,26 +17,28 @@ def build(self, hp, inputs=None):
 
 class ClassificationHead(HyperHead):
 
-    def __init__(self, **kwargs):
+    def __init__(self, binary=False, **kwargs):
         super().__init__(**kwargs)
+        self.binary = binary
         if not self.metrics:
             self.metrics = ['accuracy']
         if not self.loss:
-            self.loss = 'categorical_crossentropy'
+            if binary:
+                self.loss = 'binary_crossentropy'
+                self.output_shape = (1,)
```

Comment:
Binary classification is valid for outputs of various sizes, not just scalar outputs

Reply:
This part I don't really understand. We will discuss during the meeting.